pages/builds/parallelizing_builds.md.erb

```diff
@@ -36,11 +36,11 @@ To run the same step over all 5 of the agents we can set the parallelism field f
 
```

Comment:
So knapsack will take your test suite, divide it into parts, and run each part on different agents. It's a super neat way of parallelising tests across multiple agents.